### PR TITLE
DDF-3297 fixed performance issues with equinox

### PR DESCRIPTION
**Pull request by stustison**
#### What does this PR do?
what it says it does
#### Who is reviewing it? 
@pklinef @tbatie @rzwiefel 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl
@stustison
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by stustison** [comment]
build now

File: distribution/ddf-common/src/main/resources/common-bin.xml [hunk 1]
```diff
@@ -255,6 +256,15 @@
                 <include>org.apache.aries.blueprint.core-1.8.2.jar</include>
             </includes>
         </fileSet>
+        <fileSet>
```
Comment (coyotesqrl) [inline distribution/ddf-common/src/main/resources/common-bin.xml]: ✏️ Might want to include a comment with a link to the Eclipse ticket where the permanent fix will be tracked.

File: distribution/ddf-common/src/main/resources/common-bin.xml [hunk 2]
```diff
@@ -58,6 +58,7 @@
 
                 <!-- Skipping customized libraries. -->
                 <exclude>**/org/apache/aries/blueprint/org.apache.aries.blueprint.core/1.8.2/org.apache.aries.blueprint.core-1.8.2.jar</exclude>
+                <exclude>**/org/eclipse/platform/org.eclipse.osgi/3.11.3/org.eclipse.osgi-3.11.3.jar</exclude>
```
Comment (pklinef) [inline distribution/ddf-common/src/main/resources/common-bin.xml]: ❓ Where is the forked code at?

Comment (stustison) [reply distribution/ddf-common/src/main/resources/common-bin.xml]: https://github.com/stustison/rt.equinox.framework/commit/76f64cc71ad002d1da0c7bd6cec81afa300d6346

**Pull request merged**